composites/OnboardingWizard/components/Choice.js

```diff
@@ -65,10 +65,11 @@ const Choice = ( props ) => {
 	};
 
 	return (
-		<div className={wrapperClass}>
-			<p className="yoast-wizard-field-description">{props.properties.label}</p>
-			{fieldSet()}
-			<Explanation text={props.properties.explanation}/>
+		<div className={ wrapperClass }>
+			<p className="yoast-wizard-field-description">{ props.properties.label }</p>
+			<p> { props.properties.description } </p>
```

Comment:
Spaces outside of braces cause react to render comments in the DOM. Should be `<p>{ props.properties.description }</p>`.